<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>602</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DYNARIMA - A COVID-19 ARIMA FORECASTING TOOL</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>DYNARIMA.png</normaloff>DYNARIMA.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="btn_container">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="btn_webscrape">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Webscrape</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_compile">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Compile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_forecast">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Forecast</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCalendarWidget" name="calendar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>203</red>
                  <green>203</green>
                  <blue>203</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>125</red>
                  <green>95</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="HighlightedText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>225</red>
                  <green>225</green>
                  <blue>225</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="LinkVisited">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>29</red>
                  <green>209</green>
                  <blue>161</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="AlternateBase">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>158</red>
                  <green>158</green>
                  <blue>158</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>203</red>
                  <green>203</green>
                  <blue>203</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>125</red>
                  <green>95</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="HighlightedText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>225</red>
                  <green>225</green>
                  <blue>225</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="LinkVisited">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>29</red>
                  <green>209</green>
                  <blue>161</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="AlternateBase">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>158</red>
                  <green>158</green>
                  <blue>158</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>203</red>
                  <green>203</green>
                  <blue>203</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>56</red>
                  <green>173</green>
                  <blue>169</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="HighlightedText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>225</red>
                  <green>225</green>
                  <blue>225</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="LinkVisited">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>29</red>
                  <green>209</green>
                  <blue>161</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="AlternateBase">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>158</red>
                  <green>158</green>
                  <blue>158</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="selectedDate">
             <date>
              <year>2020</year>
              <month>1</month>
              <day>30</day>
             </date>
            </property>
            <property name="minimumDate">
             <date>
              <year>2020</year>
              <month>1</month>
              <day>30</day>
             </date>
            </property>
            <property name="dateEditEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="progressbar_container">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="progressbar_text">
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Prerequisites Initialized</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressbar">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="config_container" stretch="0,0,0,0,0,0,0,0,0,0">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="config_txt_startdate">
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="config_startdate">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="minimumDate">
               <date>
                <year>2020</year>
                <month>1</month>
                <day>30</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>MMM dd, yyyy</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="config_spacer1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>46</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="config_txt_lags">
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Lags</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="config_lags">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="config_spacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>46</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="config_txt_integrate">
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Integrate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="config_integrate">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI Variable Display</family>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="matplot_container">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="info_container">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>200</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>200</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="info_txt_adf">
          <property name="font">
           <font>
            <family>Segoe UI Variable Display</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ADF: 0.0</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="info_txt_pvalue">
          <property name="font">
           <font>
            <family>Segoe UI Variable Display</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>P-Value: 0.0</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="info_spacer_2">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="info_txt_aic">
          <property name="font">
           <font>
            <family>Segoe UI Variable Display</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>AIC: 0.0</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="info_txt_model">
          <property name="font">
           <font>
            <family>Segoe UI Variable Display</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Model: (0, 0, 0)</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="info_txt_mae">
          <property name="font">
           <font>
            <family>Segoe UI Variable Display</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>MAE: 0%</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="info_txt_accuracy">
          <property name="font">
           <font>
            <family>Segoe UI Variable Display</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Accuracy: 0%</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
